### PR TITLE
[risk=low][RW-6593] Add a Controlled Tier to Staging

### PR DESCRIPTION
**Pull request by jmthibault79**
Description:

Successfully ran: 
```
./project.rb create-auth-domain --project all-of-us-rw-staging --tier controlled --user joel-staging@staging.fake-research-aou.org
db-cdr/generate-cdr/project.rb publish-cdr --project all-of-us-rw-staging --tier controlled --bq-dataset test_R2019q4r3
```

Also renamed the "DO NOT USE" CT CDRs - we probably don't need to scare people away from these anymore

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [x] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/config/cdr_config_staging.json [hunk 1]
```diff
@@ -88,6 +96,20 @@
       "hasFitbitData": true,
       "hasCopeSurveyData": true,
       "accessTier": "registered"
+    },
+    {
+      "cdrVersionId": 7,
+      "isDefault": true,
+      "accessTier": "controlled",
+      "name": "CDR for Controlled Tier - DO NOT USE",
```
Comment (calbach) [inline api/config/cdr_config_staging.json:104]: WHy is this unsafe to use?

**Review by calbach — APPROVED** [review]

File: api/config/cdr_config_staging.json (hunk 1 above)
Comment (jmthibault79) [reply api/config/cdr_config_staging.json:104]: The original intention with these names (from Test onwards) was to indicate that it was likely to be non-functional, in the unlikely case of stumbling across it accidentally.

We can probably remove this now :)

Comment (jmthibault79) [reply api/config/cdr_config_staging.json:104]: I think I'll do that in a separate PR however

Comment (jmthibault79) [reply api/config/cdr_config_staging.json]: updated

**Review by freemabd — APPROVED** [review]
LGTM

**Pull request merged**